src/frontend/src/common/components/library/Dialog/components/DialogTitle/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { DialogTitleProps } from "@material-ui/core";
 import React from "react";
-import { StyledDialogTitle } from "./style";
+import { ExtraProps, StyledDialogTitle } from "./style";
```

Comment:
DialogTitle component now accepts an extra prop `narrow` to set width to `400px`